storagenode/peer.go

```diff
@@ -290,9 +291,21 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 		}
 		pb.RegisterPiecestoreServer(peer.Server.GRPC(), peer.Storage2.Endpoint)
 
+		sc := config.Server
+		options, err := tlsopts.NewOptions(peer.Identity, sc.Config, revocationDB)
+		if err != nil {
+			return nil, errs.Combine(err, peer.Close())
+		}
+
+		// TODO workaround for custom timeout for order sending request (read/write)
+		ordersTransport := transport.NewClientWithTimeouts(options, transport.Timeouts{
+			Dial:    20 * time.Second,
```

Comment:
Remove the 20 seconds. If the Dial timeout is not present it will use the default timeout.

Reply:
good catch, thanks